src/frequenz/sdk/actor/power_distributing/_battery_status.py

```diff
@@ -223,8 +224,80 @@ async def stop(self) -> None:
         """Stop tracking battery status."""
         await cancel_and_await(self._task)
 
-        if self._select is not None:
-            await self._select.stop()
+    def _handle_status_battery(
+        self, bat_data: BatteryData
+    ) -> None:  # Any is probably not the best
```

Comment:
I guess the comment about `Any` is outdated? Same for the rest of the new methods.

```suggestion
    ) -> None
```